### PR TITLE
Add missing header for timing target

### PR DESCRIPTION
**Pull request by dellaert**
that's it :-)

### CODE REVIEW

**Comment by dellaert** [comment]
Tagging @ayushbaid as reviewer so he knows timing code exists in GTSAM :-)

**Comment by dellaert** [comment]
@varunagrawal added you on review as well because I also removed a forgotten obsolete header that prevents double expresso from working for me. It was not needed anymore as you recall.

File: gtsam/basis/Chebyshev.h [hunk 1]
```diff
@@ -21,8 +21,6 @@
 #include <gtsam/base/Manifold.h>
 #include <gtsam/basis/Basis.h>
 
-#include <unsupported/Eigen/KroneckerProduct>
```
Comment (varunagrawal) [inline gtsam/basis/Chebyshev.h:24]: Whoops sorry about that. But great catch!

**Review by varunagrawal — APPROVED** [review]
LGTM

**Comment by varunagrawal** [comment]
Failure is because of Azure crapping out. Just need to re-run it again.

**Comment by dellaert** [comment]
OK, thanks. I'll restart that CI run.

**Comment by dellaert** [comment]
Although, that would restart a bunch. I'll just ignore that failure if all others pass.

**Comment by ayushbaid** [comment]
I did not get this. Does including the header file means we will automatically start timing all the functions in that file?

**Comment by dellaert** [comment]
> I did not get this. Does including the header file means we will automatically start timing all the functions in that file?

No, absolutely not. There are a number of timing scripts in GTSAM, but you have to enable timing with the mechanisms described in the header.

**Pull request merged**